Docker/Executor.php

```diff
@@ -336,17 +341,16 @@ public function run(Container $container, $id, $tokenInfo, $configId)
 
         // run the container
         $process = $container->run($id, $this->configData);
-        $message = $process->getOutput();
+        $message = trim($process->getOutput());
         if ($message && !$container->getImage()->isStreamingLogs()) {
             // trim the result if it is too long
             if (mb_strlen($message) > 64000) {
                 $message = mb_substr($message, 0, 32000) . " ... " . mb_substr($message, -32000);
             }
-        } else {
-            $message = "Docker container processing finished.";
+            return $message;
         }
 
-        return $message;
+        return false;
```

Comment:
Asi me prijde divny, aby run vracel false i kdyz byl uspesny, dal bych tam at to vrati prazdnej string